universe/info.owlbot/eval/scenarios.txt

```diff
@@ -0,0 +1,9 @@
+# basic
+U: Get definition of "cat"
+A: /\b(mammal)|animal|pet|creature\b
```

Comment:
I don't think you need `/` in this regular expression, and i'm not sure the parenthesis are correct. Shouldn't it be:
```
\b(mammal|animal|pet|creature)\b
```
?